packages/amplify-headless-interface/src/interface/auth/add.ts

```diff
@@ -169,10 +169,7 @@ export interface CognitoOAuthConfiguration {
   socialProviderConfigurations?: CognitoSocialProviderConfiguration[];
 }
 
-/**
- * Defines a Cognito oAuth social provider
- */
-export interface CognitoSocialProviderConfiguration {
+interface SocialProviderConfig {
```

Comment:
I think the generate schemas script needs to be run again to build from the latest TS interface

Reply:
regenerated